.github/workflows/conformance-ginkgo.yaml

```
@@ -410,6 +415,12 @@ jobs:
                -cilium.provision-k8s=false \
                -cilium.operator-suffix=-ci
 
+      - name: Get Node DMESG 
+        if: ${{ always() }}
+        shell: bash
+        run: |
+          for node in $(docker ps --format json | jq -r '.ID'); do docker exec $node dmesg ; done
```

I'd suggest getting timestamps in here too for better correlation:
```suggestion
          for node in $(docker ps --format json | jq -r '.ID'); do docker exec $node dmesg --time-format=iso; done
```

(With timestamps I don't know if we necessarily _need_ pre + post dmesg)